core/src/main/java/org/apache/calcite/rel/rules/CoreRules.java

```diff
@@ -745,6 +746,10 @@ private CoreRules() {}
   public static final UnionToDistinctRule UNION_TO_DISTINCT =
       UnionToDistinctRule.Config.DEFAULT.toRule();
 
+  /** Rule that rewrite {@link Sample} which is bernoulli to the {@link Filter}. */
+  public static final RewriteSampleToFilterRule SAMPLE_TO_FILTER =
```

Comment:
We can add this rule the the default rule sets, e.g., in `BASE_RULES`?

Reply:
yes,It makes sense for me, I will take it.